Tools/clinic/clinic.py

```diff
@@ -1988,13 +1988,30 @@ def parse_file(
     libclinic.write_file(output, cooked)
 
 
+def create_parser_namespace() -> dict[str, Any]:
+    ns = dict(
+        CConverter=CConverter,
+        CReturnConverter=CReturnConverter,
+        buffer=buffer,
+        robuffer=robuffer,
+        rwbuffer=rwbuffer,
+        unspecified=unspecified,
+    )
+    for name, converter in converters.items():
+        ns[f'{name}_converter'] = converter
+    for name, return_converter in return_converters.items():
+        ns[f'{name}_return_converter'] = return_converter
+    return ns
```

Comment:
I think we should make this a `PythonParser` method, instead of adding to the global namespace. What do you think?

Reply:
I prefer to use a function, since I reused it in eval_ast_expr().